Modules/_ctypes/stgdict.c

```diff
@@ -355,8 +355,8 @@ _ctypes_alloc_format_padding(const char *prefix, Py_ssize_t padding)
         return _ctypes_alloc_format_string(prefix, "x");
     }
 
-    int ret = PyOS_snprintf(buf, sizeof(buf), "%zdx", padding);
-    assert(0 <= ret && ret < sizeof(buf));
+    int ret = PyOS_snprintf(buf, sizeof(buf), "%zdx", padding); (void)ret;
```

Comment:
FWIW, the `(void)ret;` trick comes from the LLVM Coding Standard: https://llvm.org/docs/CodingStandards.html#assert-liberally